ksp/src/main/kotlin/fp/serrano/kopykat/CopyMap.kt

```diff
@@ -3,6 +3,7 @@
 package fp.serrano.kopykat
 
 import com.squareup.kotlinpoet.FileSpec
+import fp.serrano.kopykat.utils.*
```

Comment:
I removed these on my settings:  

![](https://user-images.githubusercontent.com/260697/192004738-fa6127da-87d8-4e99-b9db-3cf0eee85012.png)

I've created a ticket to add auto-formatting settings to the repo 🤞 #36

Saves us going back and forth 😅